### ISSUE TITLE
[EN] Group Member Person Attributes

### ISSUE THREAD

**Issue by treyhendon** [SparkDevNetwork/Rock#3007]
### Description
Several churches have requested the ability to view Person Attributes in a Group Member List, specific to the group they were viewing.

i.e. When viewing the Deacons Group, Attributes A, B, and C should be displayed/filterable and while viewing the Staff Group, Attributes B, D, E, and F should be available.

I enhanced the Group Member List to include person attributes of the group member and allow for easy selection of the desired attributes by using a Group Attribute.  The Group Attribute is of the Field Type Attribute > Person with multiple allowed.  The Group Member List block detects if any attributes like this are configured for the group type and gets the appropriate data for this group.

The person attribute is added to the grid and filters if the attribute is set to "display in grid" and if not, the value is still there an available for export to excel.  However, excel export can be disabled via block setting.

In addition, the current person's security is evaluated against the attributes - with block option to override and display.

### Affected Files
* `Rock/RockWeb/Blocks/Groups/GroupMemberList.ascx.cs`

**Comment by nairdo** [comment]
@treyhendon We reviewed this and would prefer a feature like this be done by adding a person attribute category onto the group type and/or group model.  Corresponding UI on the Group Type and Group details would let the admin choose which category they wanted to show up on the group member detail and grid. (Additionally UI mockups would need to be created before any work is done on this.)

Since you implied you've already done this work for a customer (taking a different approach), are you going to re-do this for a core pull-request, or are you going to close this issue?

**Comment by jonedmiston** [comment]
We now have a new Ideas area on our Rock Community website (https://community.rockrms.com/community) for posting, discussing and voting on enhancements to Rock. Since this issue is tagged as an enhancement we are closing it and requesting that if you still feel it has merit that you add it as an idea. This allows the full community to speak into it and help us determine it's priority.
